src/classes/Room.as

```diff
@@ -228,12 +228,9 @@ package classes
 
         public function isAllPlayersSameSong():Boolean
         {
-            if (_playerCount <= 0)
+            if (_playerCount <= 1)
                 return false;
 
-            if (_playerCount == 1)
```

Comment:
@mat100payette You may need to revert changes here when you make the solo spectating.